pingdom-checks/community-care.ping

```diff
@@ -21,6 +21,18 @@ pingdom save-check \
   -a userids_csv="$NO_USERS" \
   -a integrationids_csv="$COMMUNITY_CARE_SLACK_ID,$PAGER_DUTY"
 
+pingdom save-check \
+  --template https-public-200 \
+  -a name=production-community-care-eligibility-health \
```

Comment:
The names of these probes will need to be different, otherwise they will replace the existing probes which are named the same way. I would recommend production-cee-health .

Reply:
The previous probes actually contained a typo -> `eligibiliy` vs `eligibility` , I just used the correct spelling here. I can always update since its a bit difficult to tell the two names apart

Reply:
So we have...

current:
```
production-community-care-eligibiliy-health
production-community-care-eligibility-backend-health
```
new (proposed)
```
production-community-care-eligibility-health
production-community-care-eligibility-health-backend
```

I'm fine going with new proposed names since these probes will be basically in "shadow" mode during evaluation. @golavarr-va thoughts?

Reply:
@golavarr-va new probe names have been updated to use `cce`